src/components/Post.jsx

```diff
@@ -281,7 +281,7 @@ function Post(prop) {
         {postHasImages ? (
           <ImageSlider slides={postImages} isCommentBox={false} />
         ) : (
-          <div className="post__background">{caption}</div>
```

Comment:
it's not an addition if you're replacing the class. We still need the background from post__background class. You may add both classes and verify if it still works